### PR TITLE
Add function public function removeCheckpoints

### PR DESCRIPTION
**Pull request by Donny5300**
We wanted to disable 2 checkpoints. For that we needed to write `app('sentinel')->removeCheckpoint('checkpointKey');` twice. Thats why I forked and created a pull request

### CODE REVIEW

**Comment by brunogaspar** [comment]
Looks good, but can you please make the code consistent and compliant with PSR-2?

**Comment by Donny5300** [comment]
@brunogaspar i've took a quick lookup on the PSR-2 code standards and changed the code. Can you confirm it is correct?

**Comment by GrahamCampbell** [comment]
Still not PSR-2.

File: src/Sentinel.php [hunk 1]
```diff
@@ -629,6 +629,19 @@ public function removeCheckpoint($key)
...
+     *
+     * @param array $checkpoints
+     * @return void
+     */
+    public function removeCheckpoints( array $checkpoints = [] )
+    {
+        foreach($checkpoints as $checkpoint) {
+            $this->removeCheckpoint($checkpoint);
+        }
+	}
```
Comment (GrahamCampbell) [inline src/Sentinel.php]: Incorrect indentation.

File: src/Sentinel.php [hunk 2]
```diff
@@ -629,6 +629,19 @@ public function removeCheckpoint($key)
...
 
     /**
+     * Removes the chosen checkpoints
+     *
+     * @param array $checkpoints
+     * @return void
+     */
+    public function removeCheckpoints( array $checkpoints = [] )
+    {
+        foreach($checkpoints as $checkpoint) {
```
Comment (GrahamCampbell) [inline src/Sentinel.php]: Missing space after foreach

File: src/Sentinel.php [hunk 3]
```diff
@@ -629,6 +629,19 @@ public function removeCheckpoint($key)
     }
 
     /**
+     * Removes the chosen checkpoints
+     *
+     * @param array $checkpoints
+     * @return void
+     */
+    public function removeCheckpoints( array $checkpoints = [] )
```
Comment (GrahamCampbell) [inline src/Sentinel.php]: No spacers should be directly inside the brackets.

File: src/Sentinel.php [hunk 4]
```diff
@@ -629,6 +629,19 @@ public function removeCheckpoint($key)
     }
 
     /**
+     * Removes the chosen checkpoints
+     *
+     * @param array $checkpoints
```
Comment (GrahamCampbell) [inline src/Sentinel.php]: Need exactly 2 spaces either side of the word array.

File: src/Sentinel.php [hunk 5]
```diff
@@ -629,6 +629,19 @@ public function removeCheckpoint($key)
     }
 
     /**
+     * Removes the chosen checkpoints
```
Comment (GrahamCampbell) [inline src/Sentinel.php]: Missing fullstop at the end.

**Comment by GrahamCampbell** [comment]
I've added some detailed comments for you. :)

File: src/Sentinel.php (hunk 5 above)
Comment (Donny5300) [inline src/Sentinel.php]: Can you explain fullstop please

Comment (drsii) [inline src/Sentinel.php]: "short descriptions should end in either a full stop, exclamation mark, or question mark"

`* Removes the chosen checkpoints.`

**Pull request merged**